src/ansys/units/unit.py

```diff
@@ -8,18 +9,41 @@ class Unit:
     Parameters
     ----------
     Name: str,
-        Name of the unit or string chain of combined units.
+        Name of the unit or string chain of combined units
     Config: dict
-        dictionary of unit properties.
+        dictionary of unit properties
+    Dimensions: Dimensions
+        An instance of the Dimensions class.
+    Unit System: str
```

Comment:
I think there is a numpydoc way of stating the default here.

Also, I think you want to document the parameters as named in the constructor initializer implementation no?  i.e. it should be like this:
`unit_sys: str`